exercises/sum-of-multiples/.meta/solutions/sum_of_multiples.rb

```diff
@@ -1,10 +1,14 @@
+module BookKeeping
+  VERSION = 1
+end
+
 class SumOfMultiples
   attr_reader :multiples
   def initialize(*multiples)
     @multiples = multiples
   end
 
-  def to(limit)
+  def take(limit)
```

Comment:
`take` is also not good since we're not actually taking an array, we just want the sum.

I think I've convinced myself that the method name change was a bad idea.